playground/backend/containers/java/Dockerfile

```diff
@@ -55,6 +57,7 @@ FROM apache/beam_java8_sdk:$BEAM_VERSION
 ARG BEAM_VERSION
 ARG SPRING_VERSION=5.3.25
 ARG KAFKA_CLIENTS_VERSION=2.3.1
+ENV BEAM_VERSION=$BEAM_VERSION
```

Comment:
We should specify environment variables outside the Docker image using ConfigMap

Reply:
This is not a runtime configuration parameter in the strict sense. This variable holds the value of Beam SDK version with which the image was built. Storing this value outside of the container makes little sense to me.